docs/arcadia-users-group/20240416-intro-to-ml1/make_peptides_ml_dataset.md

```diff
@@ -0,0 +1,264 @@
+Detection Prevalence               0.1264                               0.04717              0.02453        0.03396                        0.1151          0.02075                      0.02830            0.0566                0.4472                0.1
+Balanced Accuracy                  1.0000                               0.99803              0.84211        1.00000                        0.9919          1.00000                      0.99903            1.0000                0.9894                1.0
+```
+
+## Documentation of data set contents
+
+### Column descriptions
+
+* `peptipedia_peptide_id`: cluster member peptide id.
+* `peptide_sequence`: cluster member peptide sequence.
+* `group_bioactivity`: bioactivity label that is the same between all peptides in a given cluster.
+* `all_bioactivities`: all predicted bioactivities for a given peptide.
+* `go_id`: gene ontology ID.
+* `pfam_id`: PFAM (protein family) ID.
+* `pfam_name`: name of PFAM protein family.
+* `pfam_type`: type of PFAM protein family.
+* `aliphatic_index`: relative volume occupied by aliphatic side chains (Alanine, Valine, Isoleucine, and Leucine).
+* `boman_index`: The potential interaction index proposed by Boman (2003) is an index computed by averaging the solubility values for all residues in a sequence. It can be used to give an overall estimate of the potential of a peptide to bind to membranes or other proteins. A value greater than 2.48 indicates that a protein has high binding potential.
+* `charge`: the theoretical net charge of a peptide sequence based on the Henderson-Hasselbach equation. Computed at a pH of 7 with the Lehninger pKscale.
+* `hydrophobicity`: average of the hydrophobicity values of each residue using one of the 39 scales from different sources.
+* `instability_index`: predicts the stability of a protein based on its dipeptide composition. A protein whose instability index is smaller than 40 is predicted as stable, a value above 40 predicts that the protein may be unstable.
+* `isoelectric_point`: the pH at which a particular molecule or surface carries no net electrical charge.
+* `molecular_weight`: the sum of the mass of each amino acid in the peptide sequence.
+* `pd1_residue_volume`: physical descriptor related to the peptide residue volume.
+* `pd2_hydrophilicity`: physical descriptor related to the peptide hydrophilicity.
+* `z1_lipophilicity`: Zscale quantifying the lipophilicity of the peptide.
+* `z2_steric_bulk_or_polarizability`: Zscale modeling steric properties like steric bulk and polarizability.
+* `z3_polarity_or_charge`: Zscale quantifying electronic properties like polarity and charge.
+* `z4_electronegativity_etc`: Zscale relating to electronegativity, heat of formation, electrophilicity and hardness.
+* `z5_electronegativity_etc`: Zscale relating to electronegativity, heat of formation, electrophilicity and hardness.
```

Comment:
what's the difference with the previous one?

Reply:
idk :) I chose to report zscales in this script bc the other ones (1-3) report different information than other metrics calculated by the peptides package.

If you want more information, zscales come from this publication: https://pubs.acs.org/doi/10.1021/jm9700575
and zscales documentation in the peptides package is here: https://peptides.readthedocs.io/en/stable/api.html#peptides.ZScales